metricbeat/module/vsphere/resourcepool/_meta/data.json

```diff
@@ -0,0 +1,60 @@
+{
+    "@timestamp": "2024-08-06T10:38:51.952Z",
+    "metricset": {
+        "name": "resourcepool",
+        "period": 10000
+    },
+    "service": {
+        "type": "vsphere",
+        "address": "https://localhost:8989/sdk"
+    },
+    "event": {
+        "dataset": "vsphere.resourcepool",
+        "module": "vsphere",
+        "duration": 57462824
+    },
+    "vsphere": {
+        "resourcepool": {
+            "name": "Resources",
+            "status": "green",
+            "vm": {
+                "names": [
+                  "DC0_H0_VM0", 
+                  "DC0_H0_VM1"
+                ],
+                "count": 2
+            },
+            "cpu": {
+                "active": {
+                    "average": {
+                        "pct": 0
+                    },
+                    "max": {
+                        "pct": 0
+                    }
+                },
+                "entitlement": {
+                    "mhz": 0
+                },
+                "usage": {
+                    "pct": 0,
+                    "mhz": 2018
+                }
+            },
+            "memory": {
```

Comment:
Hi, as per the datat that we listed for these metrics, looks like simulator doesn't generate these fields, but in this json I see these empty fields.
Is it like we are getting the fields but data is empty or otherwise ?

Reply:
Updated the perfMetrics to QuickStats